exporter/prometheusexporter/accumulator.go

```diff
@@ -262,7 +262,7 @@ func timeseriesSignature(ilmName string, metric pdata.Metric, attributes pdata.A
 	b.WriteString("*" + ilmName)
 	b.WriteString("*" + metric.Name())
 	attributes.Sort().Range(func(k string, v pdata.AttributeValue) bool {
-		b.WriteString("*" + k + "*" + v.StringVal())
+		b.WriteString("*" + k + "*" + pdata.AttributeValueToString(v))
```

Comment:
What about having a ToString on the value?

Reply:
I'm happy to go either ways.

Reply:
Can you do a followup PR to see how that would look like?